receiver/kafkametricsreceiver/topic_scraper.go

```diff
@@ -103,7 +106,59 @@ func (s *topicScraper) scrape(context.Context) (pmetric.Metrics, error) {
 			}
 		}
 	}
-	return s.mb.Emit(), scrapeErrors.Combine()
+
+	rb := s.mb.NewResourceBuilder()
+	rb.SetClusterAlias(s.config.ClusterAlias)
+
+	return s.mb.Emit(metadata.WithResource(rb.Emit())), scrapeErrors.Combine()
+}
+
+func (s *topicScraper) scrapeTopicConfigs(now pcommon.Timestamp, errors scrapererror.ScrapeErrors) {
+	if s.clusterAdmin == nil {
+		admin, err := newClusterAdmin(s.config.Brokers, s.saramaConfig)
+		if err != nil {
+			s.settings.Logger.Error("Error creating kafka client with admin priviledges", zap.Error(err))
```

Comment:
Out of ignorance, is there a reason this isn't being added to `errors`?

Reply:
No Particular reason, Just wanted to keep behaviour consistent with existing receiver.